assets/src/containers/AssignmentPlanning.js

```diff
@@ -84,6 +84,7 @@ function AssignmentPlanning (props) {
     if (userSettingLoaded) {
       if (isObjectEmpty(userSetting.default)) {
         setAssignmentGradeFilter(0)
+        setUserSavedFilterSetting(0)
```

Comment:
During the initial load of the assignment view, the display needs to say `My Current Setting` instead it was showing ` `Remember My Setting`, this fixes that